Cfodev 1426 cfo qa enrolment advisories to providers

vks333 opened the pull request:
Adds chart and tabular data for enrolments accepted with advisories, it displays enrolments which are accepted with advisory for provider to keep in mind for future,  service desk records the messages with prefixes 'Advisory' and 'Accepted by Exception'

carlsixsmith-moj commented:
We should not be filtering on note comments. We need to tag the notes in someway to indicate they are advisories

vks333 commented:
> We should not be filtering on note comments. We need to tag the notes in someway to indicate they are advisories

makes sense, changed the field name to "Advisory Notes" and removed sorting for this field. I have made same changes to  other QA dashboard components on the other pull request i.e. [Cfodev-1425](https://github.com/ministryofjustice/CFO-CaseAssessmentTrackingSystem/pull/907)
